roles/mongodb_auth/molecule/default/tests/test_default.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']
+).get_hosts('all')
+
+
+def include_vars(host):
+    ansible = host.ansible('include_vars',
+                           'file="../../defaults/main.yml"',
+                           False,
+                           False)
+    return ansible
+
+
+def test_mongod_cnf_file(host):
+    f = host.file('/etc/mongod.conf')
+
+    assert f.exists
+    assert f["security"]["authorization"] == "enabled"
```

Comment:
Cool! Didn't know you could do this.

Reply:
I don't know if we can... I was guessing. If we can't then I'll have to open the file to check.

Reply:
OK. Probably not then. You'll need to use contains() or check the content_string var.